src/helpers.ts

```diff
@@ -1,6 +1,8 @@
 import { api } from 'hooks/client'
 import { FileMessage, SystemMessage, UserMessage } from 'types'
 
+import { HttpResponse } from '@flowtastic/api-client';
```

Comment:
I think you should respect the prettier and eslint settings before it can be merged (no semicolon for ex.).